akka-http/src/main/scala/akka/http/javadsl/server/Route.scala

```diff
@@ -54,7 +55,8 @@ trait Route {
   def flow(system: ClassicActorSystemProvider): Flow[HttpRequest, HttpResponse, NotUsed] =
     flow(system.classicSystem, SystemMaterializer(system).materializer)
 
-  def function(system: ClassicActorSystemProvider): Function[HttpRequest, CompletionStage[HttpResponse]]
+  def function(system: ClassicActorSystemProvider): Function[HttpRequest, CompletionStage[HttpResponse]] = handler(system)
```

Comment:
`function` was only recently introduced, if we go with this, we can probably just remove it (or keep only one of `function` or `handler`.